doc/installation_instructions/platform.rst

```diff
@@ -1,30 +1,20 @@
-.. _install-notes:
+.. _platform-installation:
 
-Installation on specific platforms
-==================================
+Platform Install Guide
+======================
 
-The following describes installation details for various systems and platforms
-that SmartSim may be used on.
-
-.. include:: platform/generic.rst
-
-.. include:: platform/nonroot-linux.rst
+HPC platforms have specific modules that users can often use so they do not
```

Comment:
`HPC platforms have specific modules that users can often use so they do not need to retrieve all of the build dependencies themselves` -> `HPC platforms often provide specific modules that users can utilize to avoid retrieving all build dependencies themselves`

Reply:
i'll throw in "avoid saying users utilize" with this alternative...

`HPC platforms often provide modules that enable users to avoid retrieving all build dependencies themselves`